### PR TITLE
Added new models to tasks

### PR DESCRIPTION
**Pull request by merveenoyan**

### CODE REVIEW

File: tasks/src/text-generation/data.ts [hunk 1]
```diff
@@ -101,7 +105,7 @@ const taskData: TaskDataCustom = {
 		},
 	],
 	summary:      "Generating text is the task of producing new text. These models can, for example, fill in incomplete text or paraphrase.",
-	widgetModels: ["gpt2"],
+	widgetModels: ["bigscience/bloom-560m"],
```
Comment (osanseviero) [inline tasks/src/text-generation/data.ts:108]: I think we can use falcon here

**Review by osanseviero — APPROVED** [review]
Thanks

File: tasks/src/text-generation/data.ts (hunk 1 above)
Comment (merveenoyan) [reply tasks/src/text-generation/data.ts]: I used instruction fine-tuned 7B version as for the other ones widget was turned off (it was reason why I didn't pick it as well :/)

**Review by osanseviero — APPROVED** [review]
Thanks!

**Comment by julien-c** [comment]
hope to move away from gpt2 on https://huggingface.co/tasks/text-generation soon, it feels a bit 2019 on HF for now :)

**Pull request merged**